apb_devel/writing/reference.adoc

```diff
@@ -557,6 +557,180 @@ etherpad_db_host: "{{ lookup('env','ETHERPAD_DB_HOST') | default('mariadb', true
 state: present
 ----
 
+[[apb-devel-writing-ref-remote-clusters]]
+== Working with Remote Clusters
+
+When developing APBs, there are a few factors which could prevent the developer
+from using the full development lifecycle that the `apb` tooling offers.
+Primarily, these factors are:
+
+- Developing against an {product-title} cluster that exists on a remote host.
+- Developing APBs on a machine that does not have access to the *docker* daemon.
+
+If a developer meets any of these criteria, use the following workflow to
+publish images to the internal {product-title} registry so that the broker can
+bootstrap the image. The following sections show how to do these steps with the
+`apb` tooling and without.
+
+[[apb-devel-writing-ref-remote-clusters-pushing-apbs]]
+=== Pushing APBs
+
+To use the `apb push` command when working with a remote {product-title}
+cluster:
+
+. Ensure the base64-encoded spec is a label in the Dockerfile. This is usually
```

Comment:
s/spec/specification

Reply:
Changed to "APB spec" to be more specific.